manifests/d/drakkan/SFTPGo/2.2.0/drakkan.SFTPGo.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: drakkan.SFTPGo
+PackageVersion: 2.2.0
+PackageLocale: en-US
+Publisher: Nicola Murino
```

Comment:
```
Publisher: Nicola Murino
```
cc @drakkan: Please could you make a `Publisher:` registry key for Control Panel?

WinGet will not know where this application comes from, so people might not be able to upgrade this application.

Reply:
@ItzLevvie thanks for the review. Sorry for the trivial question, I'm far from being a windows expert :). Are you asking to add `AppPublisher` to the SFTPGo inno setup script? Actually there is [AppPublisherURL](https://github.com/drakkan/sftpgo/blob/main/windows-installer/sftpgo.iss#L23) but not `AppPublisher`. Can you please confirm? Thank you

Reply:
You can take a look at what @syvaidya did for their application: https://github.com/syvaidya/openstego/commit/7518c6bf9b69ebcb45013cd2a97188f818178a7b

Only `AppPublisher` for the Publisher to be shown or displayed in Control Panel.

Reply:
Ok, got it, thank you. I'll update this PR once I'll release SFTPGo v2.2.1 since I have to change the installer script